src/System.Drawing.Common/tests/mono/System.Drawing/ColorTests.cs

```diff
@@ -0,0 +1,183 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Drawing;
+using System.Runtime.Serialization.Formatters.Binary;
+using Xunit;
+
+namespace MonoTests.System.Drawing
+{
+    public class ColorTest
+    {
+        [Fact]
+        public void IsKnownColor()
+        {
+            Assert.True(Color.FromKnownColor(KnownColor.AliceBlue).IsKnownColor);
+            Assert.True(Color.FromName("AliceBlue").IsKnownColor);
+            Assert.Equal(false, Color.FromArgb(Color.AliceBlue.A, Color.AliceBlue.R, Color.AliceBlue.G, Color.AliceBlue.B).IsKnownColor);
+        }
+
+        [Fact]
+        public void IsNamedColor()
+        {
+            Assert.True(Color.AliceBlue.IsNamedColor);
+            Assert.True(Color.FromKnownColor(KnownColor.AliceBlue).IsNamedColor);
+            Assert.True(Color.FromName("AliceBlue").IsNamedColor);
+            Assert.Equal(false, Color.FromArgb(Color.AliceBlue.A, Color.AliceBlue.R, Color.AliceBlue.G, Color.AliceBlue.B).IsNamedColor);
+        }
+
+        [Fact]
+        public void IsSystemColor()
+        {
+            Color c = Color.FromKnownColor(KnownColor.ActiveBorder);
+            Assert.True(c.IsSystemColor);
+            Assert.True(Color.FromName("ActiveBorder").IsSystemColor);
+            Assert.Equal(false, Color.FromArgb(c.A, c.R, c.G, c.B).IsSystemColor);
+            Assert.Equal(false, Color.FromKnownColor(KnownColor.AliceBlue).IsSystemColor);
+            Assert.Equal(false, Color.FromName("AliceBlue").IsSystemColor);
+        }
+
+        [Fact]
+        public void Name()
+        {
+            Assert.Equal("AliceBlue", Color.AliceBlue.Name);
+            Assert.Equal("ActiveBorder", Color.FromKnownColor(KnownColor.ActiveBorder).Name);
+            Assert.Equal("1122ccff", Color.FromArgb(0x11, 0x22, 0xcc, 0xff).Name);
+        }
+
+        [Fact]
+        public void GetHashCodeTest()
+        {
+            Color c = Color.AliceBlue;
+            Assert.Equal(false, Color.FromArgb(c.A, c.R, c.G, c.B).GetHashCode() == c.GetHashCode());
+            Assert.Equal(c.GetHashCode(), Color.FromName("AliceBlue").GetHashCode());
+        }
+
+        [Fact]
+        public void ToArgb()
+        {
+            Assert.Equal(0x11cc8833, Color.FromArgb(0x11, 0xcc, 0x88, 0x33).ToArgb());
+            Assert.Equal(unchecked((int)0xf1cc8833), Color.FromArgb(0xf1, 0xcc, 0x88, 0x33).ToArgb());
+        }
```

Comment:
Did we not already have color tests for things like FromArgb?

Reply:
@stephentoub ouch, indeed, the whole test suit is in System.Drawing.Primitives

Reply:
Ok, thanks.  Let's please change this then to augment that suite as needed rather than duplicating.